pytest_splunk_addon_ui_smartx/components/controls/single_select.py

```diff
@@ -52,6 +53,10 @@ def __init__(self, browser, container, searchable=True, allow_new_values=False):
                 "combobox": Selector(
                     select=container.select + ' [data-test="combo-box"]'
                 ),
+                "listbox_role": Selector(select=container.select + ' [role="listbox"]'),
+                "combobox_role": Selector(
+                    select=container.select + ' [role="combobox"]'
```

Comment:
```suggestion
                    select=container.select + ' [data-test="textbox"]'
```
Please use test API of that component instead of internal implementation
https://splunkui.splunk.com/Packages/react-ui/4.16.3/ComboBox?section=test